test.sh

```diff
@@ -1,8 +1,50 @@
 #!/usr/bin/env bash
 set -e
 
-if [[ -z "$1" ]]; then
-  forge test --rpc-url "$ETH_RPC_URL"
+for ARGUMENT in "$@"
+do
+    KEY=$(echo "$ARGUMENT" | cut -f1 -d=)
+    VALUE=$(echo "$ARGUMENT" | cut -f2 -d=)
+
+    case "$KEY" in
+            match)      MATCH="$VALUE" ;;
+            block)      BLOCK="$VALUE" ;;
+            opt-block)   OPT_BLOCK="$VALUE" ;;
+            arb-block)   ARB_BLOCK="$VALUE" ;;
+            *)
+    esac
+done
+
+# L2 setup
+
+if [[ -z "$OPT_RPC_URL" ]]; then
+    echo "OPTIMISM not used"
+    export OPT_RPC_URL=""
```

Comment:
Do we need to export something that's empty?

Reply:
Yes, this was throwing errors in forge if it wasn't exported at all

Reply:
Ohh ok